### ISSUE TITLE
Role and Label missing in Dashboard screen

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#841]
**Describe the bug**
Role and Label missing for various anchor tags which are acting like buttons at Dashboard screen.

**To Reproduce**
1. Go to Dashboard screen "http://localhost:9992/cm/app/?view=dash&pathType=unknown"
2. You will see various Dashboard Gadgets which we can add on Dashboard screen.
3. If we look at "Pages By Status" Dashboard gadget, there is a "Select Filters" button .
4. Now, if we inspect that button, we will notice it is an anchor tag, and it does not have role and label.

**Expected behavior**
Role and Label should be present for anchor tags which acts like buttons.

**Additional context**
Navigate all the options available for similar issue on all the Dashboard Gadgets and fix.
